lib/X25519KeyAgreementKey2019.js

```diff
@@ -57,14 +56,7 @@ export class X25519KeyAgreementKey2019 extends LDKeyPair {
    * @returns {Promise<X25519KeyAgreementKey2019>} Generates a key pair.
    */
   static async generate(options = {}) {
-    let keyPair;
-
-    if(env.nodejs && env.hasDiffieHellman) {
```

Comment:
So `crypto.diffieHellman` is only available in node 14 and this package supports node 12. This will need a little adjusting so that the nacl stuff can be loaded in node's version as needed. So I recommend making a naclCrypto.js file that crypto-browser.js just reexports -- and crypto.js can pull in an use if `crypto.diffieHellman` is not defined.